nvme-print-stdout.c

```diff
@@ -4693,8 +4694,10 @@ static bool stdout_detailed_ns(const char *name, void *arg)
 			return true;
 		}
 
-		nvme_subsystem_for_each_ctrl(nvme_ns_get_subsystem(n), c)
+		nvme_namespace_for_each_path(n, p) {
```

Comment:
Looks good but to make sure let me confirm if is this worked for not multipath case since the function `nvme_namespace_for_each_path()` is used to check multipath as below.
```c
static bool nvme_is_multipath(nvme_subsystem_t s)
{
...
	nvme_subsystem_for_each_ns(s, n)
		nvme_namespace_for_each_path(n, p)
			return true;
...
	return false;
}
```

Reply:
In the case of !multipath, `nvme_ns_get_ctrl(n))` will return a valid pointer right above  (ca. line 4691)

Reply:
And what's worth, I've tested this for both scenarios

Reply:
Thanks for your explanations and confirmation. I could understand them.